src/components/ConnectionInterface.vue

```diff
@@ -57,6 +57,19 @@
                     <span>Save Passwords</span>
                     <i class="material-icons" v-tooltip="'Passwords are encrypted when saved'">help_outlined</i>
                   </label>
+
+                  <!-- Label Color Selection -->
+                  <div class="row">
```

Comment:
I've been thinking about the `ConnectionLabel` component.

I think for the form it would be easier to maintain if this were all just part of a `ColorPicker` component that renders all the color radios and has its own `v-model`. I think having to individually specify the colors here, all bound to the same variable makes it easy to forget one down the line if this needs to go in another place.

Then in the connection sidebar, I don't think the color needs a component, but rather a simple `span` (or whatever element) with the connection color, I think in the end it is a little weird using an `input` in the sidebar.

Reply:
Definitely makes sense. I thought it was a little awkward as well using a radio in the sidebar, this sounds much better!